src/components/Contentful/MediaItemsCentered.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<section-with-background-classic
+		:background-content="sectionBackground"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid
+					:style="customGridStyles"
+					class="tw-grid-cols-12 lg:tw-grid-cols-10 lg:tw-gap-y-4
+						tw-justify-items-center tw-text-center tw-rounded tw-bg-white
+						tw-py-5 lg:tw-px-[177px]"
```

Comment:
These bracketed tailwind classes make me squirm, since they add that class to the global stylesheet loaded on all pages even though it's only used once, but I acknowledge there's not always a better way.

Let's take a second look at this next week and see if there's not a way we can solve this with grid or flexbox.